src/BPDual.jl

```diff
@@ -17,6 +17,63 @@ end
 
 Base.lastindex(t::ASPTracer) = lastindex(t.active)
 
+
+
+@doc raw"""
+```julia
+    function bpdual(A, b, λin, bl, bu; kwargs...)``` 
+
+    Solve the optimization problem:
+
+    DP:
+    ```math 
+        \min_{y} \left( -b^T y + \frac{1}{2} λ y^T y \right)
+    ```
+        subject to:
+     ```math 
+        bl \leq A^T y \leq bu
+        ```
+
+    using given `A`, `b`, `bl`, `bu`, and `λ`. When `bl = -e` and `bu = e = ones(n, 1)`,
+    DP is the dual Basis Pursuit problem:
+
+    BPdual:
+
+    ```math 
+    \max_{y} \left( b^T y - \frac{1}{2} λ y^T y \right)
+    ```
+        subject to
+        
+    ```math
+        \|A^T y\|_{\infty} \leq 1.
+    ```
+
+    * Input
+    * `A` : `m`-by-`n` explicit matrix or operator.
+    * `b` : `m`-vector.
+    * `λin` : Nonnegative scalar.
+    * `bl`, `bu` : `n`-vectors (bl lower bound, bu upper bound).
+    * `active`, `state`, `y`, `S`, `R` : May be empty or output from `BPdual` with a previous value of `λ`.
```

Comment:
"corresponding value of"

Reply:
sorry I'm not quite sure what you mean by this :)